baldrick/plugins/github_pull_requests.py

```diff
@@ -96,11 +96,11 @@ def handle_pull_requests(repo_handler, payload, headers):
 
     return process_pull_request(
         repo_handler.repo, number, repo_handler.installation,
-        action=payload['action'], is_new=is_new)
+        action=payload['action'], is_new=is_new, payload=payload)
 
 
 def process_pull_request(repository, number, installation, action,
-                         is_new=False):
+                         is_new=False, *, payload):
```

Comment:
Does this mean `payload` has to be a keyword? If so, why is no default assigned?

Reply:
because I didn't want to change the signature, but we always should pass payload in.

Reply:
The other change is already a breaking change, so why not just change the signature here also?

Reply:
I personally think it makes sense like this though, as a required keyword argument